File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/abstractfactory/trasnport/Car.java
package com.bigboxcode.designpattern.abstractfactory.trasnport;

public class Car implements Transport {
    @Override
    public void start() {
        System.out.println("Car Started");
    }

    @Override
    public void stop() {
        System.out.println("Car Stopped");
    }

    @Override
    public void repair() {
        System.out.println("Car Repair");
    }
}

File: design-patterns/ts/factory/transport-factory-method/car.ts
// car.ts

import Transport from "../transport/transport";

class Car implements Transport {
    start(): void {
        console.log("Car started");
    }

    stop(): void {
        console.log("Car Stopped");
    }

    repair(): void {
        console.log("Car Repair");
    }
}

export default Car;